operations/finance/spending-company-money/how-to-use-accrualify/how-to-review-a-po.md

```diff
@@ -0,0 +1,41 @@
+# Reviewing a Purchase Order
+
+If you need to approve a PO, Accrualify will send you an email alerting you that a PO is in your queue for review and approval. The email will include an orange button titled “Review PO Request”. Click on the orange button and it will direct you to the PO that needs your attention.
```

Comment:
```suggestion
If you need to approve a PO, Accrualify will send you an email alerting you that a PO is in your queue for review and approval. The email will include an orange button titled “Review PO Request”. Click the orange button and it will direct you to the PO that needs your attention.
```